### PR TITLE
Switch to jbuilder

### PR DESCRIPTION
**Pull request by djs55**
Signed-off-by: David Scott <dave@recoil.org>

### CODE REVIEW

**Comment by djs55** [comment]
Maybe it's possible to avoid making the package split? I'll have a look.

**Comment by djs55** [comment]
It turns out it is possible to maintain the sub package -- I needed to specify the `(modules ( ... ))` in the `jbuild` file to override the default, where the default includes all files in the current directory. I've modified the PR.

File: Makefile [hunk 1]
```diff
@@ -4,19 +4,24 @@ TESTS = true
...
 	@
 
 depends:
 	opam pin add ${PINOPTS} prometheus .
 	opam pin add ${PINOPTS} prometheus-app .
+	opam pin add ${PINOPTS} prometheus-app-unix .
+
+prometheus-app-unix:
+	jbuilder build
+	jbuilder runtest
```
Comment (talex5) [inline Makefile]: I think you can do this in one go with `jbuild build @install @runtest`

**Review by talex5 — COMMENTED** [review]
LGTM!

**Pull request merged**